src/components/StatusDashboard/current_migrations.jsx

```diff
@@ -100,6 +102,16 @@ export default function CurrentMigrations({ onLoad }) {
             sort={state.sort.closed}
           />
         </table>
+        <table className={styles.migrations_table}>
+          <TableContent
+            collapsed={state.collapsed.paused}
+            name="Paused migrations"
+            resort={resort("paused")}
+            rows={paused}
```

Comment:
```suggestion
            rows={paused || []}
```

To avoid the `...` in the count, when it should be zero, I think?

Reply:
Hm, that didn't work. It might be somewhere else in the code then. My concern here is that we display `...` even when the (empty) file loaded. And we should distinguish from "waiting for file" (signified by `...`) and "file was received but is empty" (this should be `0`).

Reply:
Ok I will try to fix that.